examples/01_counter/tsconfig.json

```diff
@@ -8,6 +8,8 @@
     "skipLibCheck": true,
     "noUncheckedIndexedAccess": true,
     "exactOptionalPropertyTypes": true,
-    "jsx": "react-jsx"
+    "jsx": "react-jsx",
+    "outDir": "./dist",
+    "rootDir": "./src"
```

Comment:
Not sure why they are required, but maybe they are?

Reply:
it is required otherwise the output file structure will be
```
- dist
  - src
    - xxx.js
```

instead of

```
- dist
- xxx.js
```

It's not needed here since `waku build` is real output files. But just some habit for me.

But it's necessary for `packages/waku` since we're monorepo tsconfig now

Reply:
hm, that makes sense.